Sources/SwifterSwift/SceneKit/SCNVector3Extensions.swift

```diff
@@ -107,9 +109,9 @@ public extension SCNVector3 {
     ///   - vector: self.
     ///   - scalar: scalar value.
     /// - Returns: result of multiplication of the given CGPoint with the scalar.
-    static func *= (point: inout SCNVector3, scalar: SceneKitFloat) {
+    static func *= (vector: inout SCNVector3, scalar: SceneKitFloat) {
```

Comment:
This is a breaking change, should we add it in the changelog?

Reply:
Of course, will update!

Reply:
Actually not, I think the name should not make any difference here, since it is used as an operator

Reply:
Nice 👍